subprojects/emcee/queue-worker-api/src/main/kotlin/com/avito/emcee/worker/RegisterWorkerResponse.kt

```diff
@@ -16,5 +16,6 @@ public data class RegisterWorkerResponse(
         val portRange: Any,
         val payloadSignature: String,
         val maximumCacheTTL: Long,
+        val trackSigkillIgnorance: Boolean
```

Comment:
I prefer to see KDoc on this property.
Its' semantics is not clear

Reply:
It's not clear to me too. We don't use this property now, but can't skip it's declaration here, because we use `failOnUnknown` setting on Moshi configuration.
I'll plan to turn off this later and clean up on the models in a separate task later

Reply:
Could you create a task and add it number here